bd-crash-handler/src/monitor_test.rs

```diff
@@ -309,6 +319,97 @@ async fn config_file() {
   assert!(!setup.config_file_exists(INGESTION_CONFIG_FILE));
 }
 
+#[tokio::test]
+async fn test_log_report_fields() {
+  let mut builder = FlatBufferBuilder::new();
+  let name = Some(builder.create_string("BigProb"));
+  let reason = Some(builder.create_string("missing meta"));
+  let error = Error::create(
+    &mut builder,
+    &ErrorArgs {
+      name,
+      reason,
+      stack_trace: None,
+      relation_to_next: ErrorRelation::CausedBy,
+    },
+  );
+  let errors = Some(builder.create_vector::<WIPOffset<Error<'_>>>(&[error]));
+  let app_id = Some(builder.create_string("com.example.foo.widget"));
```

Comment:
I have some code internally that lets us replace this with 
```
let app_id = "com.example.foo.widget".to_fb(&mut builder)
```
We could bring this into this repo if we think it helps readability

Reply:
https://github.com/bitdriftlabs/shared-core/pull/198